diff --git a/meson.build b/meson.build
index 6d7d3ca6..1b6b4da3 100644
--- a/meson.build
+++ b/meson.build
@@ -78,11 +78,11 @@ if cc.get_id() == 'clang'
 endif
 
 if cc.get_id() == 'gcc'
-    if cc.version().version_compare('<8')
-        error('gcc version is too old, libcamera requires 8.0 or newer')
+    if cc.version().version_compare('<7')
+        error('gcc version is too old, libcamera requires 7.0 or newer')
     endif
 
-    # On gcc 8, the file system library is provided in a separate static
+    # On gcc 7 and 8, the file system library is provided in a separate static
     # library.
     if cc.version().version_compare('<9')
         cpp_arguments += [
@@ -92,8 +92,9 @@ if cc.get_id() == 'gcc'
 
     # gcc 7.1 introduced processor-specific ABI breakages related to parameter
     # passing on ARM platforms. This generates a large number of messages
-    # during compilation. Silence them.
-    if host_machine.cpu_family() == 'arm'
+    # during compilation with gcc >=7.1. Silence them.
+    if (host_machine.cpu_family() == 'arm' and
+        cc.version().version_compare('>=7.1'))
         cpp_arguments += [
             '-Wno-psabi',
         ]
