Index: pipewire/src/modules/module-filter-chain/lv2_plugin.c
===================================================================
--- pipewire.orig/src/modules/module-filter-chain/lv2_plugin.c	2022-09-02 20:14:56.000000000 +0530
+++ pipewire/src/modules/module-filter-chain/lv2_plugin.c	2022-09-03 12:06:00.061146037 +0530
@@ -35,11 +35,26 @@
 #include <pipewire/array.h>
 
 #include <lilv/lilv.h>
-#include <lv2/atom/atom.h>
-#include <lv2/buf-size/buf-size.h>
-#include <lv2/worker/worker.h>
-#include <lv2/options/options.h>
-#include <lv2/parameters/parameters.h>
+
+#if defined __has_include
+#   if __has_include (<lv2/atom/atom.h>)
+
+        #include <lv2/atom/atom.h>
+        #include <lv2/buf-size/buf-size.h>
+        #include <lv2/worker/worker.h>
+        #include <lv2/options/options.h>
+        #include <lv2/parameters/parameters.h>
+
+#   else
+
+        #include <lv2/lv2plug.in/ns/ext/atom/atom.h>
+        #include <lv2/lv2plug.in/ns/ext/buf-size/buf-size.h>
+        #include <lv2/lv2plug.in/ns/ext/worker/worker.h>
+        #include <lv2/lv2plug.in/ns/ext/options/options.h>
+        #include <lv2/lv2plug.in/ns/ext/parameters/parameters.h>
+
+#   endif
+#endif
 
 #include "plugin.h"
 
