Index: blueman-git/configure.ac
===================================================================
--- blueman-git.orig/configure.ac
+++ blueman-git/configure.ac
@@ -3,7 +3,7 @@ AC_PREREQ(2.61)
 AC_INIT([blueman], [2.3.2], [https://github.com/blueman-project/blueman/issues])
 AC_CONFIG_HEADERS(config.h)
 AC_CONFIG_MACRO_DIRS([m4])
-AM_INIT_AUTOMAKE([1.16.3 foreign dist-xz])
+AM_INIT_AUTOMAKE([foreign dist-xz])
 
 AM_MAINTAINER_MODE([enable])
 AC_PROG_CC
