Index: qpwgraph/src/qpwgraph_pipewire.cpp
===================================================================
--- qpwgraph.orig/src/qpwgraph_pipewire.cpp
+++ qpwgraph/src/qpwgraph_pipewire.cpp
@@ -303,6 +303,7 @@ static
 const struct pw_proxy_events qpwgraph_proxy_events = {
 	.version = PW_VERSION_PROXY_EVENTS,
 	.destroy = qpwgraph_proxy_destroy,
+	.bound = nullptr,
 	.removed = qpwgraph_proxy_removed,
 };
 // proxy-events.
@@ -571,6 +572,7 @@ const struct pw_core_events qpwgraph_cor
 	.version = PW_VERSION_CORE_EVENTS,
 	.info = nullptr,
 	.done = qpwgraph_core_event_done,
+	.ping = nullptr,
 	.error = qpwgraph_core_event_error,
 };
 // core-events.
@@ -612,6 +614,10 @@ void qpwgraph_link_proxy_error ( void *d
 static
 const struct pw_proxy_events qpwgraph_link_proxy_events = {
 	.version = PW_VERSION_PROXY_EVENTS,
+	.destroy = nullptr,
+	.bound = nullptr,
+	.removed = nullptr,
+	.done = nullptr,
 	.error = qpwgraph_link_proxy_error,
 };
 // link-events.
