Index: libcamera_gc52e8429/src/libcamera/meson.build
===================================================================
--- libcamera_gc52e8429.orig/src/libcamera/meson.build	2021-09-21 09:55:48.000000000 +0530
+++ libcamera_gc52e8429/src/libcamera/meson.build	2021-09-21 12:39:18.902533613 +0530
@@ -132,6 +132,7 @@
                            install : true,
                            include_directories : includes,
                            build_rpath : '/',
+						    version: '0.1.0',
                            dependencies : libcamera_deps)
 
 libcamera_public = declare_dependency(sources : [
